locations/spiders/bureau_vallee_fr.py

```
@@ -0,0 +1,17 @@
+
+
+class BureauValleeFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "bureau_vallee_fr"
+    item_attributes = {
+        "brand": "Bureau Vallée",
+        "brand_wikidata": "Q18385014",
+        "extras": Categories.SHOP_STATIONERY.value,
+    }
+    sitemap_urls = ["https://magasins.bureau-vallee.fr/sitemap_pois.xml"]
+    sitemap_rules = [(r"https://magasins\.bureau-vallee\.fr/fr/france-FR/[^/]+/[^/]+/[^/]+$", "parse_sd")]
+    wanted_types = ["LocalBusiness", "OfficeEquipmentStore"] 
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)